packages/fxa-settings/src/components/Icons/index.tsx

```diff
@@ -0,0 +1,166 @@
+import { ReactComponent as BackupRecoverySmsDisabled } from './icon_backup_recovery_sms_disabled.svg';
+import { ReactComponent as BackupRecoverySms } from './icon_backup_recovery_sms.svg';
+import { ReactComponent as CheckmarkGreen } from './icon_checkmark_green.svg';
+import { ReactComponent as CheckmarkCircleFullGreen } from './icon_checkmark_circle_full_green.svg';
+import { ReactComponent as CheckmarkCircleOutlineBlack } from './icon_checkmark_circle_outline_black.svg';
+import { ReactComponent as Code } from './icon_code.svg';
+import { ReactComponent as FlagCanada } from './icon_flag_canada.svg';
+import { ReactComponent as FlagUsa } from './icon_flag_usa.svg';
+import { ReactComponent as InformationCircleOutlineBlue } from './icon_information_circle_outline_blue.svg';
+import { ReactComponent as InformationCircleOutlineBlack } from './icon_information_circle_outline_black.svg';
+
+export const AlertIcon = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedIcon
+    Image={Alert}
+    ariaLabel="Alert"
+    ariaLabelFtlId="alert-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const AuthenticatorAppIcon = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedIcon
+    Image={AuthenticatorApp}
+    ariaLabel="Authenticator Application"
+    ariaLabelFtlId="authenticator-app-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const BackupCodesIcon = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedIcon
+    Image={BackupCodes}
+    ariaLabel="Backup codes enabled"
+    ariaLabelFtlId="backup-codes-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const BackupCodesDisabledIcon = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedIcon
+    Image={BackupCodesDisabled}
+    ariaLabel="Backup codes disabled"
+    ariaLabelFtlId="backup-codes-disabled-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const BackupRecoverySmsIcon = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedIcon
+    Image={BackupRecoverySms}
+    ariaLabel="Recovery SMS enabled"
+    ariaLabelFtlId="backup-recovery-sms-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const BackupRecoverySmsDisabledIcon = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedIcon
+    Image={BackupRecoverySmsDisabled}
+    ariaLabel="Recovery SMS Disabled"
+    ariaLabelFtlId="backup-recovery-sms-disabled-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const CheckmarkGreenIcon = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedIcon
+    Image={CheckmarkGreen}
+    ariaLabel="Check"
+    ariaLabelFtlId="checkmark-icon-aria-label"
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const CheckmarkCircleFullGreenIcon = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedIcon
+    Image={CheckmarkCircleFullGreen}
+    ariaLabel="Check"
```

Comment:
This also applies to the other checkmark icons - 'success' or 'enabled' might be more semantically correct than 'check' for a screen-reader user, but which one to use depends on context (in success banner for example vs in settings rows). Maybe we'd need a prop to allow selection of either aria-label 🤔

Reply:
Good point. I've added a mode to checkmark icons that can be enabled | success | check, and updated the parameter where needed. It seems like in most cases 'success' is the desired mode.

Reply:
This is great! Thanks for the update ❤️